release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1495,6 +1547,14 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * The ability to delete operands from the web console using the `delete all operands` option is currently disabled. It will be re-enabled in a future version of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2012120[BZ#2012120] and link:https://bugzilla.redhat.com/show_bug.cgi?id=2012971[BZ#2012971].
 
+* This release contains a known issue with Jenkins. If you customize the hostname and certificate of the OpenShift OAuth route, Jenkins no longer trusts the OAuth server endpoint. As a result, users cannot log in to the Jenkins console if they rely on the OpenShift OAuth integration to manage identity and access.
++
+Workaround: See the Red Hat Knowledgebase solution, link:https://access.redhat.com/solutions/6470721[Deploy Jenkins on OpenShift with Custom OAuth Server URL]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1991448[*BZ#1991448*])
+
+* This release contains a known issue with Jenkins. The `xmlstarlet` command line toolkit, which is required to validate or query XML files, is missing from this {op-system-base}-based image. This issue impacts deployments that do not use OpenShift OAuth for authentication. Although OpenShift OAuth is enabled by default, users can disable it.
++
+Workaround: Consider using OpenShift OAuth for authentication. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055653[BZ#2055653])
```

Comment:
```suggestion
Workaround: Use OpenShift OAuth for authentication. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055653[BZ#2055653])
```
If this is the only way to work around the issue and users need to work around it, then just tell them to use `OpenShift Oauth` rather than merely to consider using it.

Reply:
Thanks. Updated.